### PR TITLE
new port: sfml 2.4.1

### PR DESCRIPTION
**Pull request by rkitover**
C++ multimedia/network library popular with game developers and others.

Portfile based on the original submission here:

https://trac.macports.org/ticket/38987

but updated and improved.

The trac has been updated as well.

Happy to make any other improvements you would like to see to get this
merged.

### CODE REVIEW

**Comment by rkitover** [comment]
Updated to link to openal-soft instead of system OpenAL framework.

File: multimedia/sfml/Portfile [hunk 1]
```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           github 1.0
```
Comment (raimue) [inline multimedia/sfml/Portfile]: This Portfile does not seem to use the `github` port group anywhere. Just remove this line.

**Comment by rkitover** [comment]
Fixed, thanks for the quick response!

**Comment by raimue** [comment]
Merged with slight changes:
  * Fixed distname for case-sensitive filesystems
  * Rewrote commit message

**Pull request closed without merging**